setup.py

```diff
@@ -96,9 +96,9 @@ def merge_req_lists(req_lists: Iterable[List[str]]) -> List[str]:
 )
 
 test_requirements = [
-    "pytest >=7.2.1",
-    "pytest-cov >=4.0.0",
-    "pytest-asyncio>=0.14.0",
+    "pytest >=6.2.4,<7.0.0",
+    "pytest-cov >=2.12.0,<3.0.0",
+    "pytest-asyncio >=0.14.0,<0.15.0",
```

Comment:
Maybe keep upper bounds and update versions?
```suggestion
    "pytest >=7.2.1,<8.0.0",
    "pytest-cov >=4.0.0,<5.0.0",
    "pytest-asyncio >=0.14.0,<0.15.0",
```